app/src/main/kotlin/com/wire/android/ui/common/ClickableText.kt

```diff
@@ -137,7 +138,11 @@ class LinkSpannableString(source: CharSequence) : SpannableString(source) {
     }
 
     override fun setSpan(what: Any?, start: Int, end: Int, flags: Int) {
-        super.setSpan(what, start, end, flags)
-        spanList.add(Data(what, start, end))
+        if (start >= 0 && end <= this.length && start <= end) {
```

Comment:
instead of ignoring the span can we set the start to be 0 if it is less than 0 and end to be length - 1 if is is bigger than

Reply:
Good idea